### ISSUE TITLE
about the Learning rate

### ISSUE THREAD

**Issue by srtianxia** [Justin1904/Low-rank-Multimodal-Fusion#9]
In this code,  the parameters of  fusion_weights , fusion_bias is not contain in optimizer,  Don't they need to be learned? thanks !

**Comment by Justin1904** [comment]
Good point... It's been quite a while since I worked on this project. As I recall there should not be a reason to exclude those two parameters from the optimizer. This is in fact a PyTorch reimplementation of our model (which was originally written in Keras), I might've made a mistake during reimplementation.

Feel free to try to include those two parameters in the training. If you could also share your results about performance between incl. and excl. those two params I'd very much appreciate that - meanwhile, I'll try to dig a bit into our original code and see if we have this somewhat weird setting there as well.

**Comment by srtianxia** [comment]
@Justin1904 
I just want to confirm whether these two parameters need to be learned. I did not run the data in the original paper. Thanks

**Comment by Justin1904** [comment]
Yes, set them as learnable in the worst case should not hurt performance, so I recommend that.

**Comment by srtianxia** [comment]
@Justin1904  Thanks!
